misc/life_mngr/life_mngr.c

```diff
@@ -98,16 +99,19 @@ int main(int argc, char *argv[])
 	do {
 		if (node == NODE_UOS_SERVER) {
 			memset(recvbuf, 0, sizeof(recvbuf));
-			read(fd_uos, recvbuf, sizeof(recvbuf));
+			ret = read(fd_uos, recvbuf, sizeof(recvbuf));
```

Comment:
no need to check the return value?

Reply:
This "ret" is just for clear warning.
"ret" is no need to check, as the "recvbuf" is checked.